src/sentry/profiles/task.py

```diff
@@ -0,0 +1,125 @@
+from typing import Any, MutableMapping
+
+from django.conf import settings
+
+from sentry.lang.native.symbolicator import Symbolicator
+from sentry.models import Project
+from sentry.profiles.device import classify_device
+from sentry.tasks.base import instrumented_task
+from sentry.utils import json, kafka_config
+from sentry.utils.pubsub import KafkaPublisher
+
+processed_profiles_publisher = None
+
+
+@instrumented_task(  # type: ignore
+    name="profiles.process",
+    queue="profiles.process",
+    default_retry_delay=5,
+    max_retries=5,
+    acks_late=True,
+)
+def process_profile(profile: MutableMapping[str, Any]) -> None:
+    global processed_profiles_publisher
+
+    config = settings.KAFKA_TOPICS[settings.KAFKA_PROFILES]
+    processed_profiles_publisher = KafkaPublisher(
+        kafka_config.get_kafka_producer_cluster_options(config["cluster"]),
+    )
+
+    if profile["platform"] == "cocoa":
+        if not _validate_ios_profile(profile=profile):
+            return None
+        profile = _symbolicate(profile=profile)
+
+    profile = _normalize(profile=profile)
+    processed_profiles_publisher.publish(
+        "processed-profiles",
+        json.dumps(profile),
+    )
+
+
+def _normalize(profile: MutableMapping[str, Any]) -> MutableMapping[str, Any]:
+    normalized_profile = {
+        "device_locale": profile["device_locale"],
+        "device_manufacturer": profile["device_manufacturer"],
+        "device_model": profile["device_model"],
+        "device_os_name": profile["device_os_name"],
+        "device_os_version": profile["device_os_version"],
+        "duration_ns": int(profile["duration_ns"]),
```

Comment:
It is passed from the clients as string. Now, from Relay to ClickHouse, all parsers support serializing/deserializing int64 and that's why we're making it an `int` here. Since this format is not going to be expose to a browser, I think that's fine.